api/src/routing.js

```diff
@@ -181,14 +181,17 @@ app.get('/dbinfo', (req, res) => {
   proxy.web(req, res);
 });
 
-app.get(appPrefix, (req, res) => {
-  if (('deviceID' in req.query) || req.is('application/json')) {
+app.get(appPrefix, (req, res, next) => {
+  if (
+    ('deviceID' in req.query) || 
+    req.path.endsWith('/_design/medic/_rewrite/') || // when upgrading from appcache to service-workers
```

Comment:
I think this is always true. ie:

```js
const appPrefix = pathPrefix + '_design/' + environment.ddoc + '/_rewrite/';
app.get(appPrefix, (req, res, next) => {
  // should only fire when the path is exactly: "/medic/_design/medic/_rewrite/"
});
```

I think it's still ok, it just means we can get rid of the whole `if` block and just `proxy.web` every time.

Reply:
Done